### PR TITLE
organization.md file (fixes #970)

### LINKED ISSUE #970: Create an ORGANIZATION.md
à la https://github.com/servo/servo/blob/master/ORGANIZATION.md

### PR DESCRIPTION
**Pull request by ngokevin**
**Description:** To help devs know their way around A-Frame.

**Changes proposed:**
- Add organization.md file

### CODE REVIEW

File: ORGANIZATION.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - `index.js` - Main file that will be exported to `window.AFRAME`.
+  - `lib` - Integration with third-party modules.
+  - `shaders` - Standard shaders. They are required via `shaders/index.js`.
+  - `style` - Core CSS.
+  - `systems` - Standard systems. They are required via `systems/index.js`.
+  - `utils` - Utility function and modules. They should all be exposed in `utils/index.js` which is exported to `window.AFRAME.utils`.
+- `tests` - Functional and unit tests with Karma, Mocha, Sinon, and Chai.
+- `vendor` - Third-party modules not available via NPM.
+
+# Major Dependencies
```
Comment (dmarcos) [inline ORGANIZATION.md]: What about tween, webvr-polyfill and registerElement?

Comment (ngokevin) [inline ORGANIZATION.md]: ok

File: ORGANIZATION.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- [document-register-element](https://github.com/WebReflection/document-register-element) - Custom elements.
+- [three.js](https://github.com/mrdoob/three.js) - 3D library. A-Frame is a three.js framework.
+- [tween.js](https://github.com/tweenjs/tween.js/) - Animation and interpolation ibrary.
```
Comment (bryik) [inline ORGANIZATION.md]: Typo on line 28: "ibrary" -> "library"

Comment (ngokevin) [inline ORGANIZATION.md]: thanks! need to auto set spell check on in my vimrc for .md files...

File: ORGANIZATION.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  - `utils` - Utility function and modules. They should all be exposed in `utils/index.js` which is exported to `window.AFRAME.utils`.
+- `tests` - Functional and unit tests with Karma, Mocha, Sinon, and Chai.
+- `vendor` - Third-party modules not available via NPM.
+
+# Major Dependencies
+
+- [document-register-element](https://github.com/WebReflection/document-register-element) - Custom elements.
+- [three.js](https://github.com/mrdoob/three.js) - 3D library. A-Frame is a three.js framework.
+- [tween.js](https://github.com/tweenjs/tween.js/) - Animation and interpolation library.
+- [webvr-polyfill](https://github.com/borismus/webvr-polyfill) - Mobile accelerometer for rotational tracking.
```
Comment (dmarcos) [inline ORGANIZATION.md]: webvr polyfill is more than that. I would say something similar to the description on their repo: A JavaScript implementation of the WebVR spec with mobile support for cardboard like devices.

File: ORGANIZATION.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  - `shaders` - Standard shaders. They are required via `shaders/index.js`.
+  - `style` - Core CSS.
+  - `systems` - Standard systems. They are required via `systems/index.js`.
+  - `utils` - Utility function and modules. They should all be exposed in `utils/index.js` which is exported to `window.AFRAME.utils`.
+- `tests` - Functional and unit tests with Karma, Mocha, Sinon, and Chai.
+- `vendor` - Third-party modules not available via NPM.
+
+# Major Dependencies
+
+- [document-register-element](https://github.com/WebReflection/document-register-element) - Custom elements.
```
Comment (dmarcos) [inline ORGANIZATION.md]: document.registerElement polyfill?

**Comment by ngokevin** [comment]
This is ready

**Pull request merged**